package.json

```diff
@@ -71,30 +80,17 @@
   },
   "dependencies": {
     "cac": "^6.7.14",
-    "cli-highlight": "^2.1.11",
     "consola": "^3.4.2",
-    "dotenv": "^16.6.1",
-    "extract-zip": "^2.0.1",
+    "dotenv": "^17.2.3",
     "formdata-node": "^6.0.3",
     "listr2": "^8.3.3",
-    "lodash.camelcase": "^4.3.0",
-    "lodash.kebabcase": "^4.1.1",
-    "lodash.snakecase": "^4.1.1",
     "ofetch": "^1.4.1",
-    "open": "^10.2.0",
-    "ora": "^6.3.1",
-    "prompts": "^2.4.2",
-    "zod": "^3.25.76"
+    "zod": "^3.25.0 || ^4.0.0"
```

Comment:
Didn't bump the zod range at all, just expanded it.
Ran [zod-v3-to-v4](https://github.com/nicoespeon/zod-v3-to-v4) over the repo.